client/components/map/FacilityCard.tsx

```diff
@@ -1,49 +1,208 @@
-import { Flex, Text, HStack, Button } from "@chakra-ui/react";
-import { TStateFacilities } from "app-context/SheetyContext/types";
+import { Flex, Text, HStack, Button, Box, VStack, Divider } from "@chakra-ui/react";
+import { TEmptyItem, TItemSelection, TStateFacilities } from "app-context/SheetyContext/types";
 import { Image } from "@chakra-ui/react";
+import { ChevronDownIcon, ChevronUpIcon, ExternalLinkIcon } from "@chakra-ui/icons";
 import { COLORS } from "theme";
+import React, { ReactNode, useState } from "react";
+import styles from "components/home/hideScrollbar.module.css";
+import Link from "next/link";
+import { useSheetyData } from "hooks/useSheetyData";
+import { CheckIcon } from "@chakra-ui/icons";
+import { Categories } from "api/sheety/enums";
 export const FacilityCard = ({
+	items,
 	facCardDetails,
 	facCardDistance,
-	width,
-	left,
 }: {
+	items: (TItemSelection | TEmptyItem)[];
 	facCardDetails: TStateFacilities;
 	facCardDistance: number;
-	width: string;
-	left: string;
 }) => {
+	const { getItemCategory } = useSheetyData();
+
+	const TRANSLATION_DIST = -50;
+	const [translateY, setTranslateY] = useState(0);
+	const handleMovement = () => {
+		translateY === TRANSLATION_DIST ? setTranslateY(0) : setTranslateY(TRANSLATION_DIST);
+	};
+	const widths = ["86%", "50%", "40%", "25%"];
+	const lefts = ["7%", "25%", "30%", "37.5%"];
+	const bottoms = ["-27%", "-27%", "-27%", "-27%", "-25%"];
+
+	const itemsAccepted = items.filter((item) => {
+		return facCardDetails.categoriesAccepted.includes(getItemCategory(item.name));
+	});
+
+	const itemsNotAccepted = items.filter((item) => {
+		return !facCardDetails.categoriesAccepted.includes(getItemCategory(item.name));
+	});
+
+	const capitalizeFirstLetter = (word: string) => {
+		return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+	};
+
+	const categoriesProcessor = (categories: Categories[]) => {
+		const categorySplit = categories.map((category: Categories) => {
+			const words = category.split("_");
+			const capitaliseWords = words.map((word: string) => capitalizeFirstLetter(word));
+			return capitaliseWords.join(" ");
+		});
+		return categorySplit.join(", ");
+	};
+
 	return (
 		<Flex
+			paddingBottom={2}
 			position={"fixed"}
-			height="150px"
-			width={width}
-			left={left}
+			height={"55%"}
+			width={widths}
+			left={lefts}
 			bg="white"
-			zIndex={99999}
-			bottom={"80px"}
+			zIndex={1000}
+			bottom={bottoms}
 			rounded="xl"
-			flexDir={"row"}
+			flexDir={"column"}
+			transform={`translateY(${translateY}%)`}

```

Comment:
Since we have to set these in `%`. It has slightly weird behavior when on a super wide or tall display. But these should be edge cases and i can't think of a better way to handle this, just fyi.